canonical-web-and-design/maas-ui
# fix(legacy): serve the CSS in dev

## Done

- Correctly set the asset path for serving the CSS in dev.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)

### QA steps

- Load a legacy url, the CSS should load (you can confirm by view the source and clicking on maas.css).
